cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/utils/Utils.java

```diff
@@ -15,6 +15,8 @@
 
     private static String hostname = null;
 
+    private static String systemname = "cassandra-cluster";
```

Comment:
`Configuration` class is what I called 'global configuration'. It contains `hostname`, and it should contain `systemName` as well. Util is not the place for configuration parameters. Utils is used for determining `hostname` as it can be fetched from the system programmatically.